.circleci/config.yml

```diff
@@ -55,17 +56,23 @@ jobs:
               set -e
             }
 
-            # Push a tag and a latest if tagged
-            if [ -n "${CIRCLE_TAG}" ]; then
+            if [ "${DOCKER_USER}" == "" ] || [ "${DOCKER_PASS}" == "" ]; then
+              echo "Skipping Login to Dockerhub, credentials not available."
```

Comment:
These variables aren't set for PRs, but are for things that land in main. If they're not set, then we don't want to push anything.